src/app/zeko/compile_simple/compile_simple.mli

```diff
@@ -52,3 +48,16 @@ val compile :
         with type out_t = 'out_t
          and type out_var = 'out_var
          and type branches = ('first_input, 'branches) cons_branch )
+
+val is_compile_simple_real :
+  ( Proof.t * Verification_key.t
+  , Pickles.Side_loaded.Proof.t * Pickles.Side_loaded.Verification_key.t )
+  Base.Type_equal.t
+  option
+
+val add_plonk_constraint :
```

Comment:
is this being used somewhere?

Reply:
Not in this PR, should have been in the next PR maybe